test/templates/cluster_endtoend_test.tpl

```diff
@@ -79,6 +81,21 @@ jobs:
 
         # Install everything else we need, and configure
         sudo apt-get install -y percona-server-server percona-server-client make unzip g++ etcd git wget eatmydata xz-utils
+
+        {{else}}
+q
```

Comment:
The workflows are in the `expected` state, aka not running, because of this line

Reply:
I think i tried to sneak in a quick push and leave the letter here :)